index.js

```diff
@@ -97,8 +129,19 @@ module.exports.sync = function (src, dest, opts) {
 		pos += bytesRead;
 	}
 
-	const stat = fs.fstatSync(read, src);
-	fs.futimesSync(write, stat.atime, stat.mtime, dest);
-	fs.closeSync(read);
-	fs.closeSync(write);
+	fixupTimestampAndClose(read, write, src, dest);
+};
+
+module.exports.sync = function (src, dest, opts) {
+	if (!src || !dest) {
+		throw new CpFileError('`src` and `dest` required');
+	}
+
+	opts = Object.assign({overwrite: true}, opts);
+
+	if (fs.copyFileSync) {
```

Comment:
This will never be `false` since we define it in `fs.js`. You'll have to handle this logic in there.